examples/producer.go

```diff
@@ -23,7 +23,7 @@ func main() {
 	config.Net.TLS.Enable = true
 	config.Net.TLS.Config = nil
 
-	superstream.Init("token", config)
+	config = superstream.Init("token", config, superstream.ConsumerGroup("<group-name>"), superstream.Servers("<server>"))
```

Comment:
instead server you can place broker var inside.
Also add a consumer to the example and demonstrate the thing that init should be called multiple times